Package.swift

```diff
@@ -0,0 +1,60 @@
+// swift-tools-version:5.0
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import PackageDescription
+
+let package = Package(
+    name: "App Center",
+    products: [
+        .library(
+            name: "AppCenter",
+            targets: ["AppCenter"]),
+        .library(
+            name: "AppCenterAnalytics",
+            targets: ["AppCenterAnalytics"])
+    ],
+    dependencies: [
+    ],
+    targets: [
+        .target(
+            name: "AppCenter",
+            dependencies: [],
+            path: "AppCenter/AppCenter",
+            exclude: ["Support"],
+            cSettings: [
+                .define("APP_CENTER_C_NAME", to: "\"appcenter.ios\"", .when(platforms: [.iOS])),
+                .define("APP_CENTER_C_NAME", to: "\"appcenter.macos\"", .when(platforms: [.macOS])),
+                .define("APP_CENTER_C_NAME", to: "\"appcenter.tvos\"", .when(platforms: [.tvOS])),
+                .define("APP_CENTER_C_VERSION", to:"\"3.1.1\""),
```

Comment:
You don't need to keep a real version here. Below script doesn't commit this so it seems like `3.1.1` forever.
Can you just use a variable or some of meaningful string and use this in the script to replace to a real version?

Reply:
Or should it be the version that we need to update on release?

Reply:
> Or should it be the version that we need to update on release?

Yes, it must be there. A version, not placeholder.

> Below script doesn't commit this so it seems like 3.1.1 forever.

It's existing script for our current release process, it will be committed during next steps.